pkg/pipelines/eventlisteners/routes.go

```diff
@@ -0,0 +1,70 @@
+package eventlisteners
```

Comment:
This module does not have much to do with eventlisteners.

Reply:
This package is for creating an EventListener, and this creates the route to point to it.

The EventListener creates its own service, so there's no service to create a route for.